src/components/IconCircle.module.scss

```diff
@@ -0,0 +1,19 @@
+.iconCircle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
```

Comment:
It would be nice to control icon size, but we can improve this later

Reply:
Great point! A lot of these components can continue to improve. Feel free to open up tickets and push PR's for any improvements you'd like to see :)